syntax = "proto3";

message Date {
    uint32 year = 1;
    uint32 month = 2;
    uint32 day = 3;
}

message StartTime {
    uint32 hour = 4;
    uint32 minute = 5;
}

message Contest {
    string name = 1;
    Date date = 2;
    StartTime start_time = 3;
}

message Registration {
    uint32 bib_number = 1;
    repeated Contest contest = 2;
}

message Competitor {
    string lastname = 1;
    string firstname = 2;
    Date birth_date = 3;
}
